src/Microsoft.AspNet.Mvc.ViewFeatures/ViewComponents/DefaultViewComponentInvoker.cs

```diff
@@ -179,11 +157,13 @@ public IViewComponentResult InvokeSyncCore(MethodInfo method, ViewComponentConte
             {
                 _diagnosticSource.BeforeViewComponent(context, component);
                 _logger.ViewComponentExecuting(context);
-
+                var indexedArguments = ControllerActionExecutor.PrepareArguments(
```

Comment:
We should make sure to support default values as well. 

I don't know what you mean? why would null be invalid?